library/src/main/java/bio/terra/landingzone/library/configuration/stairway/LandingZoneStairwayConfiguration.java

```diff
@@ -35,6 +35,8 @@ public StairwayComponent getStairwayComponent(
     stairwayProperties.setGcpPubSubTopicId(landingZoneStairwayProperties.getGcpPubSubTopicId());
     stairwayProperties.setGcpPubSubSubscriptionId(
         landingZoneStairwayProperties.getGcpPubSubSubscriptionId());
-    return new StairwayComponent(kubeService, kubeProperties, stairwayProperties);
+
+    return new StairwayComponent(
+        kubeService, kubeProperties, stairwayProperties, stairwayProperties.stairwayExecutor());
```

Comment:
this is required due to the TCL upgrade carrying a Stairway upgrade; specifically this change: https://github.com/DataBiosphere/stairway/pull/149

Is my code change here correct to address the Stairway feature?

Reply:
That looks reasonable to me

Reply:
I haven't encountered this Stairway feature before, but agree that this approach makes sense